### ISSUE TITLE
Cannot activate only one exporter

### ISSUE THREAD

**Issue by andrei1489** [devopsfaith/krakend-opencensus#5]
I tried setting up krakend with only influxdb with the config provided by kraken designer. 
Got the following config:
```"version": 2,
  "extra_config": {
    "github_com/devopsfaith/krakend-gologging": {
      "level": "WARNING",
      "prefix": "[KRAKEND]",
      "syslog": false,
      "stdout": true
    },
    "krakendesigner": {
      "endpoint_defaults": {
        "github.com/devopsfaith/krakend-ratelimit/juju/router": {
          "maxRate": 0,
          "clientMaxRate": 0
        }
      }
    },
 "github_com/devopsfaith/krakend-opencensus": {
      "sample_rate": 100,
      "reporting_period": 1,
      "exporters": {
        "influxdb": {
          "address": "http://192.168.99.9:8086",
          "db": "krakend"
        }
      }
    }
  },
```

When I try a request through the gateway, I get the following error:
```
kraken        | panic: runtime error: invalid memory address or nil pointer dereference
kraken        | [signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0xa71e30]
kraken        | 
kraken        | goroutine 27 [running]:
kraken        | github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/exporter/jaeger.(*Exporter).ExportSpan(0x0, 0xc4202c7180)
kraken        | 	/go/src/github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/exporter/jaeger/jaeger.go:125 +0x30
kraken        | github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/trace.(*Span).End.func1()
kraken        | 	/go/src/github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/trace/trace.go:282 +0x14f
kraken        | sync.(*Once).Do(0xc4203d6308, 0xc4203e96c8)
kraken        | 	/usr/local/go/src/sync/once.go:44 +0xbe
kraken        | github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/trace.(*Span).End(0xc4203d62d0)
kraken        | 	/go/src/github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/trace/trace.go:268 +0x61
kraken        | github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/plugin/ochttp.(*bodyTracker).Read(0xc42034ace0, 0xc420318400, 0x200, 0x200, 0x7f30e8daa6c8, 0x0, 0xc42015db00)
kraken        | 	/go/src/github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/plugin/ochttp/trace.go:100 +0x135
kraken        | github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/plugin/ochttp.(*tracker).Read(0xc4200b7e60, 0xc420318400, 0x200, 0x200, 0xc4203e98f0, 0x66ee34, 0xc7ac40)
kraken        | 	/go/src/github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/plugin/ochttp/client_stats.go:108 +0x56
kraken        | github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/plugin/ochttp.(*bodyTracker).Read(0xc42034ad00, 0xc420318400, 0x200, 0x200, 0xca2500, 0x100000000000001, 0x0)
kraken        | 	/go/src/github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/plugin/ochttp/trace.go:94 +0x55
kraken        | github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/plugin/ochttp.(*tracker).Read(0xc4200b7e00, 0xc420318400, 0x200, 0x200, 0x1301900, 0xbec16983b8c9f495, 0xc4203e98c8)
kraken        | 	/go/src/github.com/devopsfaith/krakend-ce/vendor/go.opencensus.io/plugin/ochttp/client_stats.go:108 +0x56
kraken        | net/http.(*cancelTimerBody).Read(0xc42034ad20, 0xc420318400, 0x200, 0x200, 0x0, 0x200, 0x0)
....
```
It appears as though all the exporters are registered even if they are not present.

I am running krakend on docker, using a docker compose script. The docker image is latest, krakend 0.5.

I tried running with all the exporters configured and present in the docker config ( jaeger, zipkin, prometheus) and all receive data. 

Even removing the port config on the prometheus section would cause the same issue.

Is this a bug, or am I doing something wrong?

Thanks!

**Comment by taik0** [comment]
Hi @andrei1489 ,
You are totally right. This is a bug (fixed on this [commit](https://github.com/devopsfaith/krakend-opencensus/commit/1d4ab71518fa3a56ea32ded42962509ae77233b7)).
We had pending release the 0.5.1 version that fixes this problem (that we have done right now).
Please, check the [docker hub](https://hub.docker.com/r/devopsfaith/krakend/) or the rpm/deb/tgz repos from the Krakend [download page](http://www.krakend.io/download/).

**Comment by alombarte** [comment]
Thanks for reporting @andrei1489 

Is fixed now after releasing 0.5.1, closing the issue. Feel free to reopen if anything related is not quite what you expected.
